proto/bep/bep.proto

```diff
@@ -75,6 +73,18 @@ enum Compression {
   COMPRESSION_ALWAYS = 2;
 }
 
+enum FolderType {
+	FOLDER_TYPE_SEND_RECEIVE = 0;
+	FOLDER_TYPE_SEND_ONLY = 1;
+	FOLDER_TYPE_RECEIVE_ONLY = 2;
+	FOLDER_TYPE_RECEIVE_ENCRYPTED = 3;
+}
+
+enum StopReason {
```

Comment:
Imo this should get a `Folder` prefix like the type above to scope it to folders: E.g. a device might also be stopped, but for different reasons.